dev-util/lockrun/lockrun-20240415.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Lockrun - runs cronjobs with overrun protection"
```

Comment:
I know you just copied this, but we don't need `$PN` in the description...

Reply:
Should be better now, but we've used lockrun for way more than just cronjobs come to think of it, anything really that you want to protect from concurrent invocation can benefit from this if it doesn't have native locking.